scripts/cast-on-tenderly/index.js

```diff
@@ -0,0 +1,110 @@
+import 'dotenv/config';
+import axios from 'axios';
+import { Contract, ethers } from 'ethers';
+
+const NETWORK_ID = 5;
+const CHAINLOG_ADDRESS = '0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F';
+const CHIEF_HAT_SLOT = 12;
+const DEFAULT_TRANSACTION_PARAMETERS = { gasLimit: 1000000000 };
```

Comment:
Just one small nit:

```suggestion
const DEFAULT_TRANSACTION_PARAMETERS = { gasLimit: 1_000_000_000 };
```

Node.js supports numeric separators [since v12.8.1](https://node.green/#ES2021-features--numeric-separators).